planning/behavior_velocity_planner/src/scene_module/intersection/debug.cpp

```diff
@@ -150,7 +150,8 @@ visualization_msgs::msg::MarkerArray createPathMarkerArray(
 }
 
 visualization_msgs::msg::MarkerArray createVirtualStopWallMarkerArray(
-  const geometry_msgs::msg::Pose & pose, const int64_t lane_id, const std::string & stop_factor)
+  const geometry_msgs::msg::Pose & pose, const int64_t lane_id, const std::string & stop_factor,
+  const double offset_z = 0.0)
```

Comment:
Why is offset_z necessary?

Reply:
because these markers are created exactly same place and it's difficult to read message of both without offset